src/menu.css.ts

```diff
@@ -52,3 +60,29 @@ export const hideItems = style({
     opacity: 0,
     transform: 'scale(0)',
 });
+
+export const menuItem = style({
+    userSelect: 'none',
+    transition: 'background-color 0.15s ease-in-out',
+    '@media': {
+        [mq.supportsHover]: {
```

Comment:
use the same transitions as lists for hover and active states
https://github.com/Telefonica/mistica-web/blob/fcbb4a543545c21df10ccbc9537f3b5d81279b37/src/list.css.ts#L13C13-L13C61